lib/enkf/tests/enkf_local_dataset.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright (C) 2020  Equinor ASA, Norway.
+
+   The file 'enkf_local_dataset.cpp' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <stdlib.h>
+#include <stdexcept>
+#include <ert/util/test_util.hpp>
+#include <ert/enkf/local_dataset.hpp>
+
+
+void test_create() {
+  local_dataset_type * ld = local_dataset_alloc("DATA");
+  local_dataset_add_node(ld, "PERMX");
+  local_dataset_add_node(ld, "PERMY");
+  local_dataset_add_node(ld, "PERMZ");
+  test_assert_false( local_dataset_has_row_scaling(ld, "PERMX"));
+  test_assert_false( local_dataset_has_row_scaling(ld, "PERMY"));
+  test_assert_false( local_dataset_has_row_scaling(ld, "PERMZ"));
+
+  {
```

Comment:
not needed block

Reply:
Arrrrgh - "Use blocks freely" will be on my tombstone; but I will remove it for now!